src/Hangfire.Tags.PostgreSql/PostgreSqlJob.cs

```diff
@@ -0,0 +1,38 @@
+﻿// This file is part of Hangfire. Copyright � 2013-2014 Sergey Odinokov.
```

Comment:
If this file comes from the Hangfire repo can you provide a comment on the class with a link to that file?

Reply:
will add the link